src/content/en/2020/http2.md

```diff
@@ -69,48 +66,110 @@ Last year’s analysis of HTTP Archive data showed that HTTP/2 was used for over
 
- <figure markdown>
-| Protocol | Desktop | Mobile |
-| -------- | ------- | ------ |
-| HTTP/1.0 |  0.06%  |  0.05% |
-| HTTP/1.1 | 49.22%  | 50.05% |
-| HTTP/2   | 49.97%  | 49.28% |
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Protocol</th>
+        <th>Desktop</th>
+        <th>Mobile</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>HTTP/1.0</td>
+        <td class="numeric">0.06%</td>
+        <td class="numeric">0.05%</td>
+      </tr>
+      <tr>
+        <td>HTTP/1.1</td>
+        <td class="numeric">49.22%</td>
+        <td class="numeric">50.05%</td>
+      </tr>
+      <tr>
+        <td>HTTP/2</td>
+        <td class="numeric">49.97%</td>
+        <td class="numeric">49.28%</td>
+      </tr>
+    </tbody>
+  </table>
 
-<figcaption>{{ figure_link(caption="HTTP version usage for home pages.", sheets_gid="1447413141", sql_file="measure_of_all_http_versions_for_main_page_of_all_sites.sql") }}</figcaption>
+  <figcaption>{{ figure_link(caption="HTTP version usage for home pages.", sheets_gid="1447413141", sql_file="measure_of_all_http_versions_for_main_page_of_all_sites.sql") }}</figcaption>
 </figure>
 
-gQUIC is not seen in the home page data for two reasons. To measure a website over gQUIC the HTTP Archive crawl would have to perform protocol negotiation via the [alternative services](#alternative-services) header and then use this endpoint to load the site over gQUIC. This is not supported this year - expect it to be available in next year’s almanac. Also, gQUIC is predominantly used for third-party Google tools rather than serving home pages.
-
-The drive to increase security and privacy on the web has seen requests over TLS increase by over 150% in the last 4 years. Today, over 86% of all requests on mobile and desktop are encrypted. Looking just at web site home pages, the numbers are still an impressive, 78.1% of desktop and 74.7% of mobile. This is important because HTTP/2 is only supported by browsers over TLS. The proportion served over HTTP/2, as shown in Figure 22.5, has also increased 10% from last year, from 55% to 65%.
+gQUIC is not seen in the home page data for two reasons. To measure a website over gQUIC, the HTTP Archive crawl would have to perform protocol negotiation via the [alternative services](#alternative-services) header and then use this endpoint to load the site over gQUIC. This was not supported this year, but expect it to be available in next year's Web Almanac. Also, gQUIC is predominantly used for third-party Google tools rather than serving home pages.
 
-<figure markdown>
-| Protocol | Desktop | Mobile |
-| -------- | ------- | ------ |
-| HTTP/1.1 | 36.05%  | 34.04% |
-| HTTP/2   | 63.95%  | 65.96% |
+{# TODO(authors, analysts): Where does the 150% stat come from? Can we link to it or show it as a figure? #}
```

Comment:
@bazzadp @dotjs

Reply:
Is that from here @dotjs : https://httparchive.org/reports/state-of-the-web#pctHttps. Agree it should be linked.